### PR TITLE
Samples: Automatic updates to public repository

### PR DESCRIPTION
**Pull request by csu-bot-zivid**
Remember to do the following:
    1. Ensure that modified/deleted/new files are correct
    2. Make this commit message relevant for the changes
    3. Force push
    4. Delete branch after PR is merged

If this commit is an update from one SDK version to another,
make sure to create a release tag for previous version.

### CODE REVIEW

File: source/applications/advanced/color_balance.py [hunk 1]
```diff
@@ -201,6 +201,8 @@ def _color_balance_calibration(camera: zivid.Camera, settings_2d: zivid.Settings
     return (corrected_red_balance, corrected_green_balance, corrected_blue_balance)
 
 
+
+
```
Comment (SatjaSivcev) [inline source/applications/advanced/color_balance.py:205]: This should be removed from all files.

File: source/applications/advanced/create_depth_map.py [hunk 2]
```diff
@@ -53,6 +55,7 @@ def _point_cloud_to_cv_bgr(point_cloud: zivid.PointCloud) -> np.ndarray:
     return bgr
 
 
+
```
Comment (SatjaSivcev) [inline source/applications/advanced/create_depth_map.py:58]: same here (there should be two lines in total)

**Review by SatjaSivcev — CHANGES_REQUESTED** [review]
cosmetic changes needed. Can you fix @andersfagerli ?

File: source/applications/advanced/color_balance.py (hunk 1 above)
Comment (torbsorb) [reply source/applications/advanced/color_balance.py]: Simply removing these lines will cause the direct links to fail.. as they would be off by 2.

File: source/camera/advanced/capture_hdr_print_normals.py [hunk 3]
```diff
@@ -5,7 +5,6 @@
 
 """
 
-
```
Comment (SatjaSivcev) [inline source/camera/advanced/capture_hdr_print_normals.py:8]: intentionally removed @andersfagerli ?

Comment (andersfagerli) [reply source/camera/advanced/capture_hdr_print_normals.py:8]: Yes, but should fix this in SDK instead. Currently waiting on the doctag fixes before doing anything here.

**Review by TTPinto — APPROVED** [review]
LGTM

**Pull request merged**